### PR TITLE
fixed some typos & format errors in the README

### PR DESCRIPTION
**Pull request by OGtrilliams**
Fixed a couple of typos & changed formatting a bit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,8 @@
 # ttl.sh
 
-An ephemeral container registry for CI workflows.
+## An ephemeral container registry for CI workflows.
```
Comment (morningvera) [inline README.md:3]: should this be a heading considering it has nothing under it?

**Review by morningvera — APPROVED** [review]
lgtm

File: README.md (hunk 1 above)
Comment (OGtrilliams) [reply README.md:3]: I thought it looked a little better, but I can change it. the entire readme needs revamping tbqh

Comment (morningvera) [reply README.md:3]: i'm fine either way! no need to change it if it looks better 👍

**Pull request merged**